src/meson.build

```diff
@@ -248,6 +248,53 @@ if host_machine.system() == 'emscripten'
         install : true,
         install_dir : bindir,
     )
+elif host_machine.system() == 'nx'
+    taisei_elf_name = '@0@.elf'.format(taisei_basename)
+    taisei_elf = executable(taisei_elf_name, taisei_src, version_deps,
+        dependencies : taisei_deps,
+        c_args : taisei_c_args,
+        c_pch : 'pch/taisei_pch.h',
+        install : true,
+        install_dir : bindir,
+        override_options: ['strip=true'],
```

Comment:
Wasn't this supposed to be `false`?

Reply:
Most other HB projects also ship the .elf next to the nro by default, and while we don't usually bundle it into release, keeping them around is useful, unlike the nacp

Reply:
I'm confused; are you talking about the `install`? Because I was talking about the `strip=true`.